src/ui/control/fullscreen_control.js

```diff
@@ -87,12 +87,18 @@ class FullscreenControl {
 
     _setupUI() {
         const button = this._fullscreenButton = DOM.create('button', (`${this._className}-icon ${this._className}-fullscreen`), this._controlContainer);
-        button.setAttribute("aria-label", "Toggle fullscreen");
         button.type = 'button';
+        this._updateTitle();
         this._fullscreenButton.addEventListener('click', this._onClickFullscreen);
         window.document.addEventListener(this._fullscreenchange, this._changeIcon);
     }
 
+    _updateTitle() {
+        const title = this._isFullscreen() ? "Exit fullscreen mode" : "Enter fullscreen mode";
```

Comment:
Would simply "Enter fullscreen" and "Exit fullscreen" suffice?

Reply:
Yep, you are right. Fixed it.